tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -187,6 +187,13 @@ def create_bidirectional_stream(self) -> int:
         """
         stream_id = self._http.create_webtransport_stream(
             session_id=self.session_id, is_unidirectional=False)
+        # TODO(bashi): Remove this workaround when aioquic supports receiving
```

Comment:
This will be broken when the bug is fixed on aioquic. How about guarding the block with 

```
if stream.frame_type is None:
```

or 

```
if stream.frame_type is None and stream.session_id is None:
```

?

Reply:
I slightly prefer to assert because these asserts force us to remove these workarounds when we update aioquic.